### ISSUE TITLE
Animated images are jpgs

### ISSUE THREAD

**Issue by redheadanon** [Bionus/imgbrd-grabber#682]
### What steps will reproduce the problem?
Just about anything tagged "animated" on tentaclerape.net for instance will download as a .jpg

### What version of the program are you using? On what operating system?
v4.9.1  on windows 7

### Please provide any additional information below
I searched before posting this and saw that similar issues have come up in the past with previous versions. Even google chrome sometimes falls victim to this problem, so I imagine it's probably pretty annoying on your end.

Question though, is there some way for me to tell the program that if the "animated" tag exists and the filetype is .jpg .jpeg or .png that it should just use .gif instead? I tried to figure it out for a conditional name, but I imagine it would require javascript and I've never used it.

**Comment by Bionus** [comment]
I can indeed reproduce that all files end up as JPGs when downloading from that source.

However, I also tried to download the first 20 images of "animated rating:safe" in e621 and the downloaded files have the right extension:

![image](https://cloud.githubusercontent.com/assets/882719/20648575/3c2e9c10-b4ab-11e6-94e8-4dfff9eb07c3.png)

So it seems this issue is either only for Shimmie. I'm investigating and looking for a fix.

> Question though, is there some way for me to tell the program that if the "animated" tag exists and the filetype is .jpg .jpeg or .png that it should just use .gif instead? I tried to figure it out for a conditional name, but I imagine it would require javascript and I've never used it.

Actually, the "conditional filename" is pretty much integrated directly in the program. There's already a check in [image.cpp](https://github.com/Bionus/imgbrd-grabber/blob/master/src/models/image.cpp#L228-L237) that changes the extension automatically depending on the tags of the image. It has been added following the previous issues regarding animated files saving as JPGs.

As for doing it yourself, you'll need Javascript indeed, until I find a fix.
